quantum/stm32/mcuconf.h

```diff
@@ -227,7 +227,7 @@
  * SPI driver system settings.
  */
 #define STM32_SPI_USE_SPI1 FALSE
-#define STM32_SPI_USE_SPI2 FALSE
+#define STM32_SPI_USE_SPI2 TRUE
```

Comment:
I'm curious, does this impact anything by being enabled?  Eg, does it take additional resources, or increase the firmware size if it's enabled, but not actually used? 

If so, it may be worth ifdef'ing this?

Reply:
Yep, stage 2 of ChibiOS consolidation addresses this. For now, no point.
Similar example: https://github.com/tzarc/qmk_firmware/blob/platforms-chibios-refactor/keyboards/handwired/onekey/blackpill_f411/mcuconf.h

Reply:
Also, for the record, any mods to `chconf.h`/`halconf.h`/`mcuconf.h` are going to get destroyed at every ChibiOS upgrade. Thus the gymnastics in order to support this sort of conditional compilation in the consolidation work.